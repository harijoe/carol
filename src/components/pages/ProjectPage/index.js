import React from 'react'
import PropTypes from 'prop-types'

import { MainLayout, MainWrapper, InnerWrapper } from 'components'
import { ProjectDetails } from 'containers'

const ProjectPage = props =>
  <MainLayout>
    <MainWrapper>
      <InnerWrapper>
        <ProjectDetails id={props.params.projectId} />
      </InnerWrapper>
    </MainWrapper>
  </MainLayout>

ProjectPage.propTypes = {
  params: PropTypes.shape({
    projectId: PropTypes.string,
  }),
}

export default ProjectPage
