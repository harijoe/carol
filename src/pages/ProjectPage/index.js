import React from 'react'
import PropTypes from 'prop-types'

import MainLayout from 'layouts/MainLayout'
import MainWrapper from 'components/MainWrapper'
import ProjectDetails from 'containers/ProjectDetails'

const ProjectPage = props =>
  <MainLayout>
    <MainWrapper paddingTop="m">
      <ProjectDetails id={props.match.params.projectId} />
    </MainWrapper>
  </MainLayout>

ProjectPage.propTypes = {
  match: PropTypes.shape({
    params: PropTypes.shape({
      projectId: PropTypes.string,
    }),
  }),
}

export default ProjectPage
