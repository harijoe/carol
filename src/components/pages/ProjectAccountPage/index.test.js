import React from 'react'
import { shallow } from 'enzyme'
import { ThemeProvider } from 'styled-components'
import theme from '../../themes/default'
import mockIntl from '../../../../test/intlMock'

import ProjectPage from './'

it('renders ProjectPage', () => {
    expect(shallow(mockIntl(<ThemeProvider theme={theme}><ProjectPage params={{ projectId: '1' }} /></ThemeProvider>))).toMatchSnapshot()
})
