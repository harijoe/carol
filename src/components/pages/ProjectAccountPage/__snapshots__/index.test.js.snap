exports[`test renders ProjectPage 1`] = `
ShallowWrapper {
  "complexSelector": ComplexSelector {
    "buildPredicate": [Function],
    "childrenOfNode": [Function],
    "findWhereUnwrapped": [Function],
  },
  "length": 1,
  "node": <Mock>
    <Mock
        projectId="1" />
</Mock>,
  "nodes": Array [
    <Mock>
      <Mock
            projectId="1" />
</Mock>,
  ],
  "options": Object {},
  "renderer": ReactShallowRenderer {
    "_instance": ShallowComponentWrapper {
      "_calledComponentWillUnmount": false,
      "_compositeType": 2,
      "_context": Object {},
      "_currentElement": <ProjectAccountPage
        params={
                Object {
                        "projectId": "1",
                      }
        } />,
      "_debugID": 1,
      "_hostContainerInfo": null,
      "_hostParent": null,
      "_instance": StatelessComponent {
        "_reactInternalInstance": [Circular],
        "context": Object {},
        "props": Object {
          "params": Object {
            "projectId": "1",
          },
        },
        "refs": Object {},
        "state": null,
        "updater": Object {
          "enqueueCallback": [Function],
          "enqueueCallbackInternal": [Function],
          "enqueueElementInternal": [Function],
          "enqueueForceUpdate": [Function],
          "enqueueReplaceState": [Function],
          "enqueueSetState": [Function],
          "isMounted": [Function],
          "validateCallback": [Function],
        },
      },
      "_mountOrder": 1,
      "_pendingCallbacks": null,
      "_pendingElement": null,
      "_pendingForceUpdate": false,
      "_pendingReplaceState": false,
      "_pendingStateQueue": null,
      "_renderedComponent": NoopInternalComponent {
        "_currentElement": <Mock>
          <Mock
                    projectId="1" />
</Mock>,
        "_debugID": 2,
        "_renderedOutput": <Mock>
          <Mock
                    projectId="1" />
</Mock>,
      },
      "_renderedNodeType": 1,
      "_rootNodeID": 0,
      "_topLevelWrapper": null,
      "_updateBatchNumber": null,
      "_warnedAboutRefsInRender": false,
    },
    "getRenderOutput": [Function],
    "render": [Function],
  },
  "root": [Circular],
  "unrendered": <ProjectAccountPage
    params={
        Object {
            "projectId": "1",
          }
    } />,
}
`;
